guidelines/sc/22/focus-appearance.html

```diff
@@ -30,8 +30,10 @@ <h4>Focus Appearance</h4>
         <li>The focus indicator is determined by the user agent and cannot be adjusted by the author, or</li>
         <li>The focus indicator and the indicator's background color are not modified by the author.</li>
     </ul>
+  
+    <p class="note">The position, size, and shape of user interface components and sub-components is determined by its visual presentation. The visual presentation includes, but is not limited to the component’s visible content, border, component-specific background, and effects emanating from the visible component such as shadows or glow effects.</p>
```

Comment:
this sets out to initially define the position/size/shape of the UIC. but this seems to come out of the blue, as nothing up to this point talked about any of those things. perhaps it needs a slightly different intro ... something more like "What is perceived as the user interface component and any sub-components (to determine whether or not a focus indicator actually encloses it) depends on its visual presentation." or similar? Basically, need to somehow glue to together so it's clear this tries to explain what needs to be taken into consideration when determining what the focus indicator actually needs to *enclose*?